Contributing.md

```diff
@@ -80,6 +80,11 @@ and added as a committer, and be given write access to the repository.
 Committers are expected to follow this policy and continue to send pull requests, go through
 proper review, and have other committers merge their pull requests.
 
+### Active Projects and Committers
+
+- [`expressjs/response-time`](https://github.com/expressjs/response-time): @carpasse
+- [`expressjs/compression`](https://github.com/expressjs/compression): @bjohansebas
```

Comment:
What's the difference between this and `### Active Projects and Captains`? Can we merge them? Ideally there isn't two different project lists and committers.

Reply:
Based on the last discussion ([context](https://github.com/expressjs/express/pull/5768#issuecomment-2233841823)) comitters and captains are different roles (different access level and responsibilities). Also currently we manage that in different Github teams.

Maybe we can use the same project list and add a sub-list in each project for the captains and comitters. So we duplicate less. But not sure if that solves the thing :thinking:

Reply:
Got it, I think it makes sense for them to be separate. Should this be a list we commit or can we manage it in GitHub and make the teams public?

If you really want to commit it, maybe move it below the current active project list and make it a sub-title like `Active Committers`? Having two sections in a long document calls "active projects" is my main confusion, and committing committers or putting them first makes less sense as I assume this document is the one which we expect people to attend to when they need to contact the DRI for a project.

Reply:
I see more collaborators in the list here that aren't being committed: https://github.com/orgs/expressjs/teams.

Reply:
We probably want to do a clean up of the teams/members :thinking:

Reply:
I will mark this as resolved with my latest changes, please re-open otherwise.